nchelpers/__init__.py

```diff
@@ -1700,15 +1705,17 @@ def _cmor_type_filename_components(
         elif self.is_other:
 
             # CAUTION: Temporary solution to a bigger problem here
```

Comment:
Let's add a link to https://pcic.uvic.ca/confluence/display/CSG/Indexing+Gridded+Observation+Datasets here so that we can skip the search phase next time this kind of problem raises its head.